README.md

```diff
@@ -66,7 +66,7 @@ PRs are welcome. Please visit [#174](https://github.com/jarun/Buku/issues/174) f
 - Portable, merge-able database to sync between systems
 - Multithreaded full DB refresh
 - Shell completion scripts, man page with handy examples
-- Colored output
+- Lightweight, clean interface, custom colors
```

Comment:
`Lightweight, clean interface` is already at the top of the feature list. Please append custom colors to it instead of a new entry at the end.